config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: gemcutter_production
```

Comment:
We don't use actioncable, do we need this file? Better to remove and keep things simple or leave it because it's vanilla rails?